src/lib/utils/hooks/SynapseAPI/file/useFiles.ts

```diff
@@ -32,3 +32,31 @@ export function useGetPresignedUrlContent(
     },
   )
 }
+
+export function useGetProfileImage(
```

Comment:
```suggestion
/**
 * Get a blob containing the image data for the avatar of a Synapse user. Returns null if the user does not have a profile image.
 */
export function useGetProfileImage(
```

Reply:
Nice comment, recommend this is committed.